cfme/tests/test_rest.py

```diff
@@ -235,7 +257,7 @@ def test_delete_arbitration_settings(request, rest_api, from_detail):
             collection.action.delete(*response)
 
 
-@pytest.mark.uncollectif(lambda: version.current_version() < '5.7')
+@pytest.mark.uncollectif(lambda: current_version() < '5.7')
```

Comment:
@psav I am thinking we should have a decorator or a marker for this particular usage :)